app/experimenter/experiments/serializers/design.py

```diff
@@ -323,7 +323,7 @@ def update(self, instance, validated_data):
             )
             for preference_data in preferences_data:
                 preference_data["experiment"] = instance
-                RolloutPreference.objects.create(**preference_data)
+                RolloutPreference(**preference_data).save()
```

Comment:
I think we should try to stick to using the [queryset manager methods](https://docs.djangoproject.com/en/3.0/ref/models/querysets/#update-or-create) where possible.  This technically works but it's kind of a hidden behaviour.